Create ui-c8y-1024-10-0-share-dashboards-across-applications-graft-release-cd.md (#4964)
SoftwareAG/c8y-docs: content/change-logs/device-management/ui-c8y-1024-10-0-share-dashboards-across-applications-graft-release-cd.md

---
date: ""

> title: Share dashboards across applications

Reviewer: ```suggestion
title: Share type dashboards across applications
```

product_area: Device management & connectivity
change_type:
  - value: change-QHu1GdukP
    label: Feature
component:
  - value: component--KIsStyzM
    label: Device Management app
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: DM-6516
version: 1024.10.0
---

> In the Dashboard Manager tab, type dashboards can now be shared across applications. Use the new Data sharing action to pull in dashboards from other applications, or the Manage visibility row action to push a dashboard to specific applications.

Reviewer 2: ```suggestion
In the dashboard manager, type dashboards can now be shared across applications. Use the new **Data sharing** option to pull in dashboards from other applications, or the **Manage visibility row** option to push a dashboard to specific applications.
```

> In the Dashboard Manager tab, type dashboards can now be shared across applications. Use the new Data sharing action to pull in dashboards from other applications, or the Manage visibility row action to push a dashboard to specific applications.

Reviewer 2: @dafkk Could you provide a screenshot from the dashboard manager? I can´t see any in the user documentation. Perhaps it is worth adding at least one.

Reviewer: Yes yes, we have a task to add a documentation update, I am currently doing that :)

> Dashboards are often used across multiple applications within your tenant, but previously you had to duplicate dashboards manually to use them in different applications. Now you can share dashboards across applications, allowing you to create a dashboard once and reuse it in multiple applications without duplication. This reduces maintenance overhead and ensures consistency across your applications, as any updates to a shared dashboard are automatically reflected in all applications that use it.

Reviewer: ```suggestion
In the Dashboard Manager tab, type dashboards can now be shared across applications. Use the new Data sharing action to pull in dashboards from other applications, or the Manage visibility row action to push a dashboard to specific applications.
```
